packages/react/src/components/card-analytics/TickerCard.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Do we need import react here?

Reply:
Yes, because of the following eslint rule: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/react-in-jsx-scope.md